psh/test-ps.py

```diff
@@ -43,6 +43,9 @@ def harness(p):
                 task = 'psh'
             if task in expected_tasks:
                 expected_tasks.remove(task)
+            # to provide compatibility with older kernel versions, where it's [idle] not [kthread]
+            for task in ('[kthread]', '[idle]'):
```

Comment:
uh, overwriting the `task` from outer scope, this will essentially remove `ktread` and `init` even only if `psh` is present

Reply:
Right - I missed it, thanks for the suggestion